assets/init

```diff
@@ -43,6 +43,14 @@ GITLAB_BACKUPS=${GITLAB_BACKUPS:-disable}
 GITLAB_BACKUP_TIME=${GITLAB_BACKUP_TIME:-04:00}
 GITLAB_BACKUP_EXPIRY=${GITLAB_BACKUP_EXPIRY:-}
 
+if [ "${GITLAB_BACKUP_PROVIDER}" == "AWS" ]; then
+  echo "Configuring AWS backups for S3 butket: ${GITLAB_BACKUP_AWS_S3_BUCKET:-}"
```

Comment:
@andycmaj you have a typo here (butket) and I think you intend to use `${GITLAB_BACKUP_AWS_S3_BUCKET}` and not `${GITLAB_BACKUP_AWS_S3_BUCKET:-}`